src/gemdat/trajectory.py

```diff
@@ -401,3 +428,51 @@ def split(self,
             ]
 
         return subtrajectories
+
+    @property
+    def MSD(self) -> np.ndarray:
```

Comment:
Just a nitpick, it's convention to only use capitals for global values. I also think it's clearer what this does if you spell out and turn it into a function.

```suggestion
    def mean_square_displacement(self) -> np.ndarray:
```

Reply:
Makes sense! Thanks